xbmc/guilib/guiinfo/GUIInfoColor.cpp

```diff
@@ -17,13 +17,19 @@
 
 using namespace KODI::GUILIB::GUIINFO;
 
-bool CGUIInfoColor::Update()
+bool CGUIInfoColor::Update(const CGUIListItem* item /* = nullptr */)
```

Comment:
Compilation failing for me in Linux due to missing includes:

```
#include "guilib/GUIListItem.h"
#include "FileItem.h"
```

Reply:
thanks